tests/unit/models/gpflow/test_models.py

```diff
@@ -1547,19 +1557,19 @@ def test_sparse_variational_trajectory_sampler_has_correct_samples(
     x_predict = tf.constant([[1.0], [2.0], [3.0], [1.5], [2.5], [3.5]], gpflow.default_float())
     x_predict_parallel = tf.expand_dims(x_predict, -2)  # [N, 1, D]
     x_predict_parallel = tf.tile(x_predict_parallel, [1, num_samples, 1])  # [N, B, D]
-    samples = trajectory(x_predict_parallel)  # [N, B, 1]
-    sample_mean = tf.reduce_mean(samples, axis=1)  # [N, 1]
-    sample_variance = tf.math.reduce_variance(samples, axis=1)  # [N, 1]
+    samples = trajectory(x_predict_parallel)  # [N, B, P]
```

Comment:
not sure if important but we use L, not P, in the model code...

Reply:
Done